### PR TITLE
Add the Chinese version of README

### PR DESCRIPTION
**Pull request by OAKChina**

### CODE REVIEW

**Comment by Luxonis-Brandon** [comment]
Thanks for doing all these!

File: collecting-training-data/README.md [hunk 1]
```diff
@@ -1 +1,36 @@
-Moved to https://github.com/luxonis/depthai-experiments/tree/master/gen2-collecting-training-data
\ No newline at end of file
+[中文文档](README.zh-CN.md)
```
Comment (VanDavv) [inline collecting-training-data/README.md:1]: This script was moved [here](https://github.com/luxonis/depthai-experiments/tree/master/gen2-collecting-training-data) - could you update the README there? It also have a different content as new flags were added

File: gen2-palm-detection/main.py [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+
+
+class Main(DepthAI):
+    def __init__(self, file=None, camera=False):
+        self.cam_size = (128, 128)
+        super().__init__(file, camera)
+
+    def create_nns(self):
+        self.create_nn(
+            "models/palm_detection_openvino_2020.1_4shave.blob", "palm", first=True
```
Comment (VanDavv) [inline gen2-palm-detection/main.py:388]: Can you compile the model to the latest 2021.3 OpenVINO? We have recently released a [blobconverter](https://pypi.org/project/blobconverter/) tool that can help you in compiling it faster

File: gen2-palm-detection/main.py [hunk 3]
```diff
@@ -0,0 +1,468 @@
...
+        print("Creating pipeline...")
+        self.pipeline = depthai.Pipeline()
+
+        if self.camera:
+            # ColorCamera
+            print("Creating Color Camera...")
+            self.cam = self.pipeline.createColorCamera()
+            self.cam.setPreviewSize(self._cam_size[1], self._cam_size[0])
+            self.cam.setResolution(
+                depthai.ColorCameraProperties.SensorResolution.THE_4_K
```
Comment (VanDavv) [inline gen2-palm-detection/main.py:106]: 1080p is enough here
```suggestion
                depthai.ColorCameraProperties.SensorResolution.THE_1080_P
```

File: gen2-palm-detection/main.py [hunk 4]
```diff
@@ -0,0 +1,468 @@
...
+            for obj in det
+            if obj[-1] > min_score_thresh
+        ]
+
+        self.palm_confs = [
+            obj[-1] for det in detections for obj in det if obj[-1] > min_score_thresh
+        ]
+
+        if len(self.palm_coords) == 0:
+            return False
```
Comment (VanDavv) [inline gen2-palm-detection/main.py:448]:
```suggestion
            return
```

**Review by VanDavv — CHANGES_REQUESTED** [review]
Thanks, @OAKChina! Saw some minor issues, but other than that, I think you've done great work here!

**Comment by OAKChina** [comment]
Hi @VanDavv 
Thank you for your review, we are not familiar with github pull requests, please correct us if we did something wrong.

**Review by VanDavv — APPROVED** [review]
Thanks @OAKChina, it looks good now - will merge

**Pull request merged**